### PR TITLE
[Feature] ResNet Strikes Back.

### PR DESCRIPTION
**Pull request by RangiLyu**
# Transfer learning of paper 'Resnet strikes back: An improved training procedure in timm'

paper URL https://arxiv.org/pdf/2110.00476

<div align=center>

</div>

# Results 

## Faster R-CNN

|    Backbone     | Lr schd | bbox mAP|
| :-----------------: | :--------: | :--------------: |
|    R50              |     1x     |        37.4      |
|    R50 (Resnet strikes back pretrain)   |     1x     |        40.8 (+3.4)     |

## Mask R-CNN

|    Backbone     | Lr schd | bbox mAP  | mask mAP |
| :-----------------: | :--------: | :--------------: | :--------------: |
|    R50              |     1x     |     38.2         |        34.7       |
|    R50 (Resnet strikes back pretrain)   |     1x     |   41.2 (+3.0)    |     38.2 (+3.5)        |

## Cascade Mask R-CNN

|    Backbone     | Lr schd | bbox mAP  | mask mAP |
| :-----------------: | :--------: | :--------------: | :--------------: |
|    R50              |     1x     |     41.2        |      35.9        |
|    R50 (Resnet strikes back pretrain)   |     1x     |  44.8 (+3.6)    |      39.9 (+4.0)     |

## RetinaNet

|    Backbone     | Lr schd | bbox mAP|
| :-----------------: | :--------: | :--------------: |
|    R50              |     1x     |    36.5     |
|    R50 (Resnet strikes back pretrain)   |     1x     |    39.0 (+2.5)    |

### CODE REVIEW

File: configs/resnet_strikes_back/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# ResNet strikes back: An improved training procedure in timm
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
```
Comment (jbwang1997) [inline configs/resnet_strikes_back/README.md:6]: The format of configs has changed.

File: configs/resnet_strikes_back/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+```latex
+@article{wightman2021resnet,
+title={Resnet strikes back: An improved training procedure in timm},
+author={Ross Wightman, Hugo Touvron, Hervé Jégou},
+journal={arXiv preprint arXiv:2110.00476},
+year={2021}
+}
+```
+
```
Comment (ZwwWayne) [inline configs/resnet_strikes_back/README.md:31]: Add some notes, e.g., we have done grid search on xxx and xxx and find which hyper-parameter works best.

File: configs/resnet_strikes_back/metafile.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+
+Collections:
+  - Name: ResNet strikes back
+    Metadata:
+      Training Data: COCO
```
Comment (jbwang1997) [inline configs/resnet_strikes_back/metafile.yml:5]: The "In Collection" needs to be "Faster R-CNN". Only "Algorithm" has a corresponding "Collections".

**Comment by xiaohu2015** [comment]
@RangiLyu very nice work! I have found that you use AdamW optimizer with the TIMM ResNet model, do you test with SGD?

**Comment by RangiLyu** [comment]
> @RangiLyu very nice work! I have found that you use AdamW optimizer with the TIMM ResNet model, do you test with SGD?

Yes, but we found that the results of SGD are unsatisfactory. One possible reason may be because the TIMM ResNet is trained with LAMB, which is an improved version of the Adam series.

**Review by ZwwWayne — APPROVED** [review]
